src/components/YITHPlugins.js

```diff
@@ -8,82 +8,83 @@ import { useEffect, useState } from "@wordpress/element";
 import apiFetch from "@wordpress/api-fetch";
 import { NewfoldRuntime } from "../sdk/NewfoldRuntime";
 import { YithFeatureCard } from "./YithFeatureCard";
-import filter from "../icons/brands/yith-woocommerce-ajax-product-filter.svg";
-import search from "../icons/brands/yith-woocommerce-ajax-search.svg";
-import booking from "../icons/brands/yith-woocommerce-booking.svg";
-import customizeAccount from "../icons/brands/yith-woocommerce-customize-myaccount-page.svg";
-import gift from "../icons/brands/yith-woocommerce-gift-card.svg";
-import wishList from "../icons/brands/yith-woocommerce-wishlist.svg";
-import { 
-  YITH_WOOCOMMERCE_ACCOUNT_PAGE,
-  YITH_WOOCOMMERCE_AJAX_PRODUCT_FILTER,
-  YITH_WOOCOMMERCE_BOOKING_APPOINTMENTS,
-  YITH_WOOCOMMERCE_GIFT_CARDS,
-  YITH_WOOCOMMERCE_WISHLIST
-} from "../constants";
+import lightchest from '../icons/light-chest.svg';
+import { LoadingPanel } from "./LoadingPanel";
 
 export function YITHPlugins({ woo, wpModules }) {
-  const yithPluginsMap = new Map([
+  const [isOpen, setIsOpen] = useState(false);
+  const [pluginName, setPluginName] = useState("");
+  const [yithProducts, setYithProducts] = useState([]);
+  let anyPluginActive = 0;
+  const [yithPluginsMap, setYithPluginsMap] = useState(new Map([
     [
-      "fc4cbc14-470d-471b-b448-c6666e5b763d",
+      "f7834881-f5df-43ab-9c7e-c4e6969f5606",
       {
-        title: "nfd_slug_yith_woocommerce_booking",
-        name: "YITH Booking and Appointment for WooCommerce",
-        learnMore: YITH_WOOCOMMERCE_BOOKING_APPOINTMENTS,
-        image: booking,
+        title: "nfd_slug_yith_woocommerce_gift_cards",
+        primaryUrl: ""
       },
     ],
     [
-      "e307cb8f-24b5-46e1-81e3-83de32c62c78",
+      "93c942e4-36fb-46be-867b-5f0d014adb22",
       {
-        title: "yith-woocommerce-ajax-search",
-        name: "YITH WooCommerce AJAX Search",
-        image: search,
+        title: "nfd_slug_yith_woocommerce_wishlist",
+        primaryUrl: ""
       },
     ],
     [
-      "93c942e4-36fb-46be-867b-5f0d014adb22",
+      "wondercart",
```

Comment:
everywhere we're using id and here wondercart, is this expected ?

Reply:
@manikantakailasa ^

Reply:
this is not yith plugin so we dont have specific id like yith